src/3rd_party_repos.c

```diff
@@ -34,27 +34,27 @@
 
 char *third_party_get_bin_dir(void)
 {
-	return sys_path_join(globals_bkp.path_prefix, SWUPD_3RD_PARTY_BIN_DIR);
+	return sys_path("%s%s", globals_bkp.path_prefix, SWUPD_3RD_PARTY_BIN_DIR);
```

Comment:
I would prefer to always use "%s/%s" to make sure we will never end up merging 2 paths without separators. sys_path is going to remove that anyway, so it's fine.

Reply:
good point